docs/products/flink/howto/connect-bigquery.md

```diff
@@ -20,9 +20,21 @@ sink using [Aiven client](/docs/tools/cli) and [Aiven Console](https://console.a
 ## Prerequisites
 
 -   Aiven for Apache Flink service
--   A Google Cloud Platform (GCP) account.
+-   Google Cloud Platform (GCP) account
```

Comment:
not related to this line but could you fix the meta description while we're in this file? :)